pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[tool.jupytext]
+formats = "ipynb,py:percent"
+notebook_metadata_filter="kernelspec,jupytext,jekyll"
```

Comment:
```suggestion
notebook_metadata_filter="jupytext,jekyll"
```

Reply:
I tried that last week and again, and it didn't work. It keeps saving the `kernelspec`. Would you mind giving it a try and see whether it works for you? There may be a flag somewhere that I'm missing.

Reply:
the trick was using `-kernelspec`

Reply:
Oh nice!